models/articles.models.js

```diff
@@ -1,5 +1,28 @@
 const db = require("../db/connection.js");
 
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT
+        a.author,
+        a.title,
+        a.article_id,
+        a.topic,
+        a.created_at,
+        a.votes,
+        a.article_img_url,
+        COUNT(comments.comment_id) as comment_count
+      FROM 
+        articles AS a 
+        LEFT JOIN comments ON a.article_id = comments.comment_id
```

Comment:
nearly there but this join is incorrect:
by joining where article_id = comment_id you are not counting how many comments contain the same article_id, you are counting how many comment_ids have the same number as the given article_id which will only ever be one